### ISSUE TITLE
Show dates in date format instead of decimal date

### ISSUE THREAD

**Issue by jotasolano** [seattleflu/simulated-data#1]
For the `/simulatedSubjectDatabase.csv` and perhaps also for the incidence model, it would be ideal for JS parsing to have dates, such as the column `timeInfected` in the format:

`YYYY-MM-DD`, for example:

`2018-03-23`

**Comment by trvrb** [comment]
In auspice JSONs, eg http://data.nextstrain.org/zika_tree.json, we distinguish between `date` which is always `YYYY-MM-DD`, eg `2006-03-16` and `num_date` which is decimal format, eg `2006.20266`.

**Comment by jotasolano** [comment]
@trvrb are you suggesting we have both columns in the data then?

**Comment by jotasolano** [comment]
Also, JavaScript counts months from 0 to 11. It's fine by me if you prefer to use months 1 to 12, just let us know what the encoding is!

**Comment by jameshadfield** [comment]
From the ISO standard: "[MM] indicates a two-digit month of the year, 01 through 12. [DD] indicates a two-digit day of that month, 01 through 31" so we should use this :)

**Comment by famulare** [comment]
I added two columns to simulatedSubjectDatabase.csv and the auspiceReady metadata tsv files.  
1) `encountered_date` in YYYY-MM-DD with MM = [01,12] and DD = [01,31].  That key name matches the format in the real data from Thomas. 
2) I copied `timeInfected` into `num_date` for compatibility with auspice.    I kept `timeInfected` in the files too so as not to immediately break anyone's prototypes.

I'll make sure this is standard going forward: https://github.com/seattleflu/incidence-mapper/issues/8

**Comment by famulare** [comment]
I added two columns to simulatedSubjectDatabase.csv and the auspiceReady metadata tsv files.

- `encountered_date` in YYYY-MM-DD with MM = [01,12] and DD = [01,31]. That key name matches the format in the real data from Thomas.
- I copied `timeInfected` into `num_date` for compatibility with auspice. I kept `timeInfected` in the files too so as not to immediately break anyone's prototypes.

I'll make sure this is standard going forward: https://github.com/seattleflu/incidence-mapper/issues/8
